src/components/icon-text/IconText.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { IconAlertCircle } from 'hds-react';
+import './IconText.scss';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
This eslint disable might be unnecessary now that there's 2 exports in the file?

Reply:
I will remove this

Reply:
- [x] removed